### PR TITLE
rosserial_leonardo_cmake: 0.1.4-1 in 'melodic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by civerachb-cpr**
Increasing version of package(s) in repository `rosserial_leonardo_cmake` to `0.1.4-1`:

- upstream repository: https://github.com/clearpathrobotics/rosserial_leonardo_cmake.git
- release repository: https://github.com/clearpath-gbp/rosserial_leonardo_cmake-release.git
- distro file: `melodic/distribution.yaml`
- bloom version: `0.10.1`
- previous version for package: `null`

## rosserial_leonardo_cmake

```
* Updated arduino link from google code
* Contributors: Dave Niewinski
```

### CODE REVIEW

**Review by hidmic — COMMENTED** [review]
@civerachb-cpr it'd be good to include a LICENSE file at the root of the source repository.

**Comment by civerachb-cpr** [comment]
Noted.  I've added the file, but I'm not sure it's worth re-blooming for that single change; the existing Kinetic release is also missing that file.

**Comment by hidmic** [comment]
Releasing tags that have a LICENSE file would be best :)

**Comment by civerachb-cpr** [comment]
Given that this is the exact same tag that was previously released for Kinetic, and the Kinetic version is also lacking that file, is the lack of the license file actively going to cause problems for this PR?

**Review by hidmic — APPROVED** [review]
@civerachb-cpr ok, I'll let this one in. Hopefully things fall in place by the time this package gets re-released.

Though I will point out, as @cottsay pointed out to me, that there are MPL-licensed files in https://github.com/clearpathrobotics/rosserial_leonardo_cmake/, not only BSD-licensed ones. Both package.xml and LICENSE file (or several of these) should reflect this.

**Review by clalancette — CHANGES_REQUESTED** [review]
As @hidmic pointed out, there are both MPL-licensed files and (implicitly) BSD-licensed files in the source repository.  Please add a `<license>MPL</license>` tag to the package.xml, add an additional LICENSE file to the top-level, and do a new release.  That should unblock this package.

**Comment by ivanpauno** [comment]
@civerachb-cpr friendly ping, could you address the previous review feedback? thanks!

**Comment by civerachb-cpr** [comment]
Sorry, I've been busy with other tasks.  I've talked to the maintainer of the package in questions and let them know about your request.

**Comment by civerachb-cpr** [comment]
I think things should be ok now; the license data has been updated & included in the 0.5.1 release.  I've bumped the version in this PR accordingly.

**Review by ivanpauno — APPROVED** [review]

**Comment by ivanpauno** [comment]
Holding for Melodic [sync](https://discourse.ros.org/t/preparing-for-melodic-sync-2021-03-15/19458).

**Review by clalancette — APPROVED** [review]

**Comment by furukawahiromi** [comment]
I'm sorry to make a mistake in the link.

**Pull request merged**
